l2geth/consensus/clique/clique.go

```diff
@@ -249,7 +249,7 @@ func (c *Clique) verifyHeader(chain consensus.ChainReader, header *types.Header,
 	}
 	number := header.Number.Uint64()
 
-	if vm.UsingOVM {
+	if !vm.UsingOVM {
```

Comment:
This seems like a non fatal bug

Reply:
Hmmm... can we leave this specific diff as part of a different PR?

Reply:
Thats fair, can remove this diff